frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -716,7 +718,10 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
           ? immutableFormData.setIn(['metadata', 'namespace'], match.params.ns).toJS()
           : immutableFormData.toJS(),
       )
-        .then(() => history.push(next))
+        .then((res) => {
+          postFormCallback(res);
```

Comment:
I realize now that we probably want the `postFormCallback` to be async so that we can ensure it's complete before executing `history.push`